config/config.go

```diff
@@ -223,6 +223,10 @@ func (c *Config) IsProFeatureEnabled() bool {
 		return true
 	}
 
+	if len(c.External["platform"]) > 0 {
```

Comment:
What happens if c.External is nil?

Reply:
If a map is nil, it will be treated as an empty map for the lookup. And if the key does not exist Go returns the zero value which is nil for `platform` and `len(nil)` being 0.

Reply:
uh, I didn't know that, it's just writing that makes it panic

Reply:
Yeah, really had to google and even then check in the playground 😅 https://go.dev/play/p/Sb6k7ApVEOx